modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/facet/FacetProcessor.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface FacetProcessor<T> {
 
+	public String getFacetClassName();
```

Comment:
@W2jx @dantewang We now prefer to use property to avoid activation of the real service instance, see https://issues.liferay.com/browse/LPS-180239. 

Please resend.

Thanks.